doc/source/text.rst

```diff
@@ -168,28 +168,34 @@ Extracting Substrings
 
 .. _text.extract:
 
-The method ``extract`` (introduced in version 0.13) accepts `regular expressions
-<https://docs.python.org/2/library/re.html>`__ with match groups. Extracting a
-regular expression with one group returns a Series of strings.
+.. versionadded:: 0.13.0
 
-.. ipython:: python
+.. warning::
+
+   In version 0.18.0, ``extract`` gained the ``expand`` argument. When
+   ``expand=False`` (default) it returns a ``Series``, ``Index``, or
+   ``DataFrame``, depending on the subject and regular expression
```

Comment:
In any case, it makes the dimension of the output perfectly predictable: `expand=False` -> 1D return, `expand=True` -> 2D return, and not dependent on the content of the regex as is now the case.